CyberHealth/assessment/views.py

```diff
@@ -8,14 +7,24 @@
 from .forms import AnswerForm
 import logging
 
-
 logger = logging.getLogger(__name__)
 
-
 @basic_auth_required
 def assessment_start_page(request):
-
-    return render(request, 'assessment/index.html')
+    logger.info(request)
+    questions = Question.objects.filter()
+    for question in questions:
+        if question.answer_set.all().first() is None:
+            question.chosen_answer = "None"
+            question.answer_colour = "blue"
+        else:
+            question.chosen_answer = question.answer_set.all().first().choice.choice_text
+            if question.chosen_answer == "yes":
+                question.answer_colour = "green"
+            else:
+                question.answer_colour = "red"
```

Comment:
I see what you mean about doing this business logic here + passing to template!

This looks good though. Is working as expected for first answer only (which I presume is intended - e.g. once I have answered "yes" to a question, if i change to no, the `assessment_start_page` doesn't update to reflect most recent answer).

Reply:
I was wrong on this one - and therefore have made a change to reflect that. It should be the latest answer `last` , not the oldest answer `first`.

Great spot

Reply:
This is working for me now, nice work!